tests/lib/screenshot-testing/run-tests.js

```diff
@@ -43,9 +43,10 @@ async function main() {
 
     mocha = new Mocha({
         ui: 'bdd',
-        reporter: config.reporter,
         bail: false,
-        useColors: true,
+        reporter: config.reporter,
+        reporterOptions: config.reporterOptions,
```

Comment:
with the newly added config condition the reporterOptions may be undefined from there
```suggestion
        reporterOptions: config.reporterOptions || {},
```

Reply:
The options will now be always exported from the config instead of having to check if the exports are complete.